src/analysis-paths.ts

```diff
@@ -2,22 +2,46 @@ import * as core from '@actions/core';
 
 import * as configUtils from './config-utils';
 
+function isInterpretedLanguage(language): boolean {
+  return language === 'javascript' || language === 'python';
+}
+
+// Builds an environment variable suitable for LGTM_INDEX_INCLUDE or LGTM_INDEX_EXCLUDE
+function buildIncludeExcludeEnvVar(paths: string[]): string {
+  return paths.filter(p => p.indexOf('**') === -1).join('\n');
```

Comment:
```suggestion
  return paths.filter(p => p.indexOf('*') === -1).join('\n');
```
Paths with either `*` or `**` make no sense in the include/exclude env variables.

Reply:
Thanks. You're right and this now checks for any instances of `*`.